products/compute/terraform.yaml

```diff
@@ -1360,6 +1360,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         exclude: true
       tunnels: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
+  VpnGateway: !ruby/object:Overrides::Terraform::ResourceOverride
+    name: 'HaVpnGateway'
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "ha_vpn_gateway_basic"
+        primary_resource_id: "ha_gateway1"
+        vars:
+          ha_vpn_gateway1_name: "ha-vpn-1"
+          network1_name: "network1"
+          ha_vpn_gateway2_name: "ha-vpn-2"
+          network2_name: "network2"
+    properties:
```

Comment:
If you want you could also add a `validateGCPName` validatefunc on the name field